pkg/jx/cmd/import.go

```diff
@@ -948,7 +949,7 @@ func (options *ImportOptions) ensureDockerRepositoryExists() error {
 		dockerRegistry := cm.Data["docker.registry"]
 		if dockerRegistry != "" {
```

Comment:
I would be nice to refactor this out in a generic interface such as `RegistryCreator` which can be implemented for different cloud providers.

Reply:
Agreed, however this function has been here before my PR. So I propose to introduce `RegistryCreator` in a dedicated PR. Sounds good? :)

Reply:
Sounds good! It doesn't need to be fixed in this PR.